### ISSUE TITLE
An option to filter out "hello" and "test" snaps

### ISSUE THREAD

**Issue by NullNoname** [bhdouglass/uappexplorer#99]
I think an option to hide all these "hello" and "test" snaps that floods the application list (https://uappexplorer.com/snaps?type=application) would be nice.
Most of these snaps are basic "hello world" apps and we probably don't need so many duplicates.

**Comment by Aury88** [comment]
95 test and 81 hello...some with the same application name (I don't know how it is possible). probably they came from [here](https://docs.snapcraft.io/build-snaps/publish) after doing [this](https://docs.snapcraft.io/build-snaps/your-first-snap)...imho it's a snapcraft documentation fault...they need do add some line about unpublish app (a simple  `snapcraft close <snap_name> <channel>` command). their last tutorial is about using the dashboard in which seem there is not an easy way to remove a snap

**Comment by bhdouglass** [comment]
Oh wow, I didn't realize there were so many, I'll see about filtering them out of the results by default.

**Comment by Aury88** [comment]
some of the test seem to be effective application needed for testing things (plus my favorite opensource minecraft alternative is named mine**test**). I've already write about this in their forum

**Comment by Aury88** [comment]
ok, the forum talk I've opened about this problem is [here](https://forum.snapcraft.io/t/publish-your-snap-tutorial-dont-cover-unpublish/3522) with 59 views but 0 replies....probably a more "official" request or some word in that talk could help unlock the situation ;-)

**Comment by NullNoname** [comment]
I think this situation won't be improved anytime soon (@Aury88's forum thread still has no reply) so I think, at the very least, applications that contain "hello" in its name or "GNU hello" in its description should be filtered-out by default in uApp Explorer website. As @Aury88 pointed out, several "test" apps are useful apps so I think they are okay to be visible by default.

**Comment by Aury88** [comment]
@NullNoname, you are right.... probably my forum was too much cryptic or considered not relevant...also applying my suggestion can only limit the new test/hello entry and probably the already uploaded hello/test snaps will remain anyway.

**Comment by bhdouglass** [comment]
I think the approach that makes sense to me is to add a new filter that hides all the test/hello apps (off by default) and let users decide if they want to turn it on or not.
